src/files/disable-intranet-compatibility-mode-in-ie.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+REG ADD "HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\BrowserEmulation" /t REG_DWORD /v IntranetCompatibilityMode /d 0 /f
```

Comment:
At the current moment IE11 is available only on Win10 and Win8.1 configurations, it works well there. About previous versions of IE and Win - it doesn't make effect, because they haven't this option.